modules/photo/include/opencv2/photo.hpp

```diff
@@ -694,7 +694,7 @@ class CV_EXPORTS_W MergeMertens : public MergeExposures
 @param exposure_weight well-exposedness measure weight
  */
 CV_EXPORTS_W Ptr<MergeMertens>
-createMergeMertens(float contrast_weight = 1.0f, float saturation_weight = 1.0f, float exposure_weight = 0.0f);
+createMergeMertens(float contrast_weight = 1.0f, float saturation_weight = 1.0f, float exposure_weight = 1.0f);
```

Comment:
Could you please revert this change, because it breaks source code compatibility (and breaks tests).

If current default parameters make no sense, you can mark this to be fixed in the next major release (current we don't maintain "next" branch):
```
#ifndef OPENCV_ABI_COMPATIBILITY
CV_EXPORTS_W Ptr<MergeMertens>
createMergeMertens(float contrast_weight = 1.0f, float saturation_weight = 1.0f, float exposure_weight = 1.0f);
#else
CV_EXPORTS_W Ptr<MergeMertens>
createMergeMertens(float contrast_weight = 1.0f, float saturation_weight = 1.0f, float exposure_weight = 0.0f);
#endif
```